### ISSUE TITLE
Add support of `if / else / end`

### ISSUE THREAD

**Issue by Flipez** [Flipez/rocket-lang#71]
syntax should look like:

```js
if(false)
  puts("if")
elif(5 < 4)
  puts("else if")
else
  puts("else")
end
```

**Comment by Flipez** [comment]
```js
a = 2
if (a == 1)
  puts("true2")
else if (a == 3)
  puts("false2")
else if (a == 2)
  puts(2)
end
```

is already supported

**Comment by MarkusFreitag** [comment]
Should `el` replace `else` in general or just be added as an alias?

**Comment by Flipez** [comment]
Should be
```
if(false)
  puts("if")
elif(5 < 4)
  puts("else if")
else
  puts("else")
end
```

**Comment by Flipez** [comment]
Implemented by #198
